test/locators.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { browser } from 'protractor';
+import { PersonalInformationPage } from '../src/page';
+
+describe('Practice automation', () => {
+  beforeAll(async () => {
+    await browser.get('https://www.tutorialspoint.com/selenium/selenium_automation_practice.htm');
+  });
+
+  describe('Fill form', () => {
+    const personalInformationPage = new PersonalInformationPage();
+
+    beforeAll(async () => {
```

Comment:
this beforeAll isn't necessary

Reply:
Protractor tells me that it is necessary and that some methods require to be executed asynchronously, and Alejo also uses it.

![image](https://user-images.githubusercontent.com/38733280/102396131-01ac6e80-3faa-11eb-893e-b26174dc23bc.png)